### PR TITLE
Update README for release 3.1.0

### PR DESCRIPTION
**Pull request by ghost**
This is will be merged after release 3.1.0

Preview : https://github.com/checkout/checkout-3ds-sdk-ios/blob/1ba4d2ca8b21679297334215b4dd13d282a0ceb6/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,8 +51,7 @@ platform :ios, '12.0'
 use_frameworks!
 
 target '<Your Target Name>' do
-  pod 'CheckoutEventLoggerKit', '1.2.3'
```
Comment (deepesh-vasthimal-cko) [inline README.md:40]: Why is CheckoutEventLoggerKit removed ?

File: README.md [hunk 2]
```diff
@@ -37,8 +51,7 @@ platform :ios, '12.0'
...
-  pod 'Checkout3DS', :git => 'git@github.com:checkout/checkout-3ds-sdk-ios.git', :tag => '3.0.0'
+  pod 'Checkout3DS', :git => 'git@github.com:checkout/checkout-3ds-sdk-ios.git', :tag => ''
```
Comment (deepesh-vasthimal-cko) [inline README.md:54]: Should this be 3.1.0 ?

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,28 +1,42 @@
...
+- Supports **3D Secure protocols 2.1.0 and 2.2.0.**
+- Presents **native 3DS2 challenge screens to the user, with styling that you can customise** to best fit your apps and branding.
+- Optionally **falls back to 3DS1** when 3DS2 is not available, returning the same authentication result to your app as for 3DS2. (This option can be configured by Checkout.com for your account.)
+- Supports a **range of local languages and accessibility needs**, and allows you to set your own string translations.
+- Meets requirements from **EMVCo and the PCI Security Standards Council**, specifically set for 3DS SDKs, so you can be sure it is interoperable with card issuers and that your customers’ sensitive data stays secure.
 
 ## Minimum Requirements
-The 3DS SDK for iOS requires Xcode 13.1 and above with Swift version 5.6.2 and above, and supports apps targeting iOS 12.0 and above. It also supports Objective-C. There are two ways to integrate our 3DS SDK.
+The 3DS SDK for iOS requires Xcode 13.1 and above with Swift version 5.6.2 and above, and supports apps targeting iOS 11.0 and above. It also supports Objective-C. There are two ways to integrate into our 3DS SDK.
```
Comment (okhan-okbay-cko) [inline README.md:35]: I think we shouldn't add `into` here. It changes the meaning in a way as if merchants integrate their apps into our SDK, rather the opposite.

File: README.md [hunk 4]
```diff
@@ -92,42 +110,43 @@ let authenticationParameters = AuthenticationParameters(
...
 ```swift
    private var transaction: Transaction?
    private var standalone3DSService: ThreeDS2Service?
 ````
 
-2. Initialize the SDK with your preferred user interface options:
+2-Initialize the SDK with your preferred user interface options:
 
-- `scheme` can be set only as `visa` and `mastercard` in lowercase string.
+- `scheme` can be set only as `visa` and `mastercard` in lower case string.
```
Comment (okhan-okbay-cko) [inline README.md:149]: Internet says both versions are right but I haven't seen anyone writing `lower case`. [Reference](https://english.stackexchange.com/a/59411/443406)

File: README.md [hunk 5]
```diff
@@ -169,23 +188,26 @@ do {
...
+      case .success(let authenticationResult):
+         // handle authentication result. Checkout Payment Authorisation section.
       case .failure(let error):
          // handle failure scenario
        }
       // call close and cleanUp methods after challenge flow is completed. 
-      self.transaction?.close()
-      self.standalone3DSService?.cleanUp()
+      self?.transaction?.close()
+      self?.standalone3DSService?.cleanUp()
```
Comment (okhan-okbay-cko) [inline README.md:210]: nit: I think it's better practice to keep a strong reference to self after it being weakified into the closure. We had some discussions around this with Alex before he left but can discuss again in our next meeting. If it's a strong reference, then we get to make sure that these two lines either run together or fail together. Currently, in a theoretical case of self getting deallocated in between these two lines, program gets half run and might go into an unknown state. Not too important though.

File: README.md [hunk 6]
```diff
@@ -216,5 +252,5 @@ Our iOS SDK depends on some external libraries:
 
 
 ## License
-This software is released under license. See [LICENSE](https://github.com/checkout/checkout-3ds-sdk-ios/blob/main/LICENSE.md) for details.
+This software is released under license. See [LICENSE](https://github.com/checkout/checkout-3ds-sdk-ios/blob/develop/LICENSE.md) for details.
```
Comment (okhan-okbay-cko) [inline README.md:255]: Is this intentional? Shouldn't it be main?

**Review by okhan-okbay-cko — APPROVED** [review]
Just added some optional comments. Feel free to disregard them please.

Other than that, looks good 🔥

File: README.md (hunk 3 above)
Comment (patrick-hoban-cko) [reply README.md:35]: Personally, I don't `into` is necessary here for it to make grammatical sense

File: README.md (hunk 4 above)
Comment (patrick-hoban-cko) [reply README.md:149]: Let's go with `lowercase`

**Review by patrick-hoban-cko — APPROVED** [review]

**Comment by patrick-hoban-cko** [comment]
Happy, just please check Okhan/Deepesh's comments around eventLoggerKit, versioning etc

**Review by okhan-okbay-cko — APPROVED** [review]
🗒️

**Pull request merged**